DBSCE/Overview.md

```diff
@@ -0,0 +1,283 @@
+
+### Cleanup of the binding keys and their artifacts
+
+For the health of the operating system and user privacy, it is important to clean up binding keys and their artifacts. If the number of keys grows too high, the performance of the OS can degrade.
+
+The cleanup can occur:
+
+- When cookies are expired.
+- On demand, when the user decides to clear browser cookies.
+- Automatically, when a key hasn't been used for a long time (N days) to keep the OS healthy.
```

Comment:
let's remove when a key hasn't been used for a long time (N days) to keep the OS healthy, and tie cleanup to cookie lifetime.

Reply:
Done.